examples/readiness-probe-pod

```diff
@@ -0,0 +1,19 @@
+# Copy of pod.yaml without file extension for test
```

Comment:
Also, all of `test/images` is included in a separate place: https://github.com/kubernetes/kubernetes/blob/master/build/common.sh#L943-L946

(We probably should not be shipping all of `test/images` in the test tarball...)